api/v2/utils.go

```diff
@@ -205,3 +210,65 @@ func (api *API) extractPostForms(c *gin.Context, formNames ...string) map[string
 	}
 	return forms
 }
+
+// Unzip will decompress a zip archive, moving all files and folders
+// within the zip file (parameter 1) to an output directory (parameter 2).
+// from https://golangcode.com/unzip-files-in-go/
+func Unzip(src string, dest string) ([]string, error) {
```

Comment:
is this safe, to unzip arbitrary zip files - or files that the uploader claim is a zip file - onto our servers?

Reply:
There are a few security issues, I'm going to add a few more checks and balances here. The main issue I knew of was zip slip which this protects against, however there are some additional issues I didnt think of.

Reply:
One of the things we might need to change is running the API docker using [gvisor](https://github.com/google/gvisor)